tests/karma-phantomjs-polyfill.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+// Basic polyfills for APIs which are supported natively
+// by all browsers we support (IE >= 10)
+require('core-js/es5');
```

Comment:
Turns out that we don't actually need `ES5` polyfills any more, I think that's because of the Phantom 2.x upgrade.